log_extractor/log_extractor.py

```diff
@@ -482,9 +491,25 @@ def parse_logs(self):
         "team it will parse log for all teams"
     )
 )
-def run(job, build, folder, logs, team):
+@click.option(
+    "--skip-download", is_flag=True, cls=helper.MutuallyExclusiveOption,
+    mutually_exclusive=["job", "build"],
+    help=(
+        "In case of pre-downloaded logs we can skip download."
+        "Must be used with local-log-file option."
+    )
+)
+@click.option(
```

Comment:
I think it can be local-log-directory
The same under help description
```
we can provide local file with logs
```
looks strange :smile:

Reply:
What about setting default '"DEFAULT=/$HOME/art-tests-logs/{job}/{build}"' since we store them in default location?
or even better, check if we have logs under the default directory or automatically skip the download if we found it.

Reply:
ok

Reply:
Regarding logs directory.. I am expecting the compressed file.. otherwise, if files are allready extracted I have to do changes in the extract_all method, because apart from extracting there, you do some copying of relevant files.